drivers/wdm/audio/backpln/portcls/undoc.cpp

```diff
@@ -98,7 +92,6 @@ PcHandlePropertyWithTable(
     // store device descriptor
     KSPROPERTY_ITEM_IRP_STORAGE(Irp) = (PKSPROPERTY_ITEM)SubDeviceDescriptor;
 
-
```

Comment:
readd this newline, no need for cosmetic changes here.

Reply:
Fixed.